### PR TITLE
zib-CareTeam

### PR DESCRIPTION
**Pull request by pieter-edelman-nictiz**
~~Geblokkeerd vanwege #41~~

### CODE REVIEW

File: resources/zib/zib-CareTeam.xml [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+    </element>
+    <element id="CareTeam.name">
+      <path value="CareTeam.name" />
+      <short value="CareTeamNaam" />
+      <definition value="The name of the care team." />
+      <alias value="ZorgTeamNaam" />
+      <mapping>
+        <identity value="zib-careteam-v1.0-2020EN" />
+        <map value="NL-CM:17.3.12" />
+        <comment value="CareTeamNaam" />
```
Comment (Ashwin-nictiz) [inline resources/zib/zib-CareTeam.xml:56]: I think this is a mistake in the zib this must be "CareTeamName"

File: resources/zib/zib-CareTeam.xml [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+        <discriminator>
+          <type value="profile" />
+          <path value="member.resolve()" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+    </element>
+    <element id="CareTeam.participant:contactperson">
+      <path value="CareTeam.participant" />
+      <sliceName value="contactperson" />
```
Comment (Ashwin-nictiz) [inline resources/zib/zib-CareTeam.xml:71]: I think this value must be "contactPerson"

File: resources/zib/zib-CareTeam.xml [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+    <element id="CareTeam.participant:healthcareProfessional.role:healthProfessionalRole">
+      <path value="CareTeam.participant.role" />
+      <sliceName value="healthProfessionalRole" />
+      <short value="HealthProfessionalRole" />
+      <definition value="The role the health professional fulfils in the healthcare process. For health professionals, this could be for example attender, referrer or performer." />
+      <alias value="ZorgverlenerRol" />
+      <max value="1" />
+      <binding>
+        <strength value="required" />
+        <description value="The role the health professional fulfils in the healthcare process." />
```
Comment (Ashwin-nictiz) [inline resources/zib/zib-CareTeam.xml:131]: I thought we didn't need to fill in the description of a value list?

File: resources/zib/zib-CareTeam.xml [hunk 4]
```diff
@@ -0,0 +1,173 @@
...
+  <kind value="resource" />
+  <abstract value="true" />
+  <type value="CareTeam" />
+  <baseDefinition value="http://hl7.org/fhir/StructureDefinition/CareTeam" />
+  <derivation value="constraint" />
+  <differential>
+    <element id="CareTeam">
+      <path value="CareTeam" />
+      <short value="CareTeam" />
+      <definition value="Root concept of the CareTeam information model.This root concept contains all data elements of the CareTeam information model." />
```
Comment (Ashwin-nictiz) [inline resources/zib/zib-CareTeam.xml:40]: This can be removed

File: resources/zib/zib-CareTeam.xml (hunk 2 above)
Comment (pieter-edelman-nictiz) [reply resources/zib/zib-CareTeam.xml:71]: I agree, but zib says no.

File: resources/zib/zib-CareTeam.xml (hunk 1 above)
Comment (pieter-edelman-nictiz) [reply resources/zib/zib-CareTeam.xml:55]: Yeah, the zib makes a mistake here. Fixed it.

File: resources/zib/zib-CareTeam.xml (hunk 3 above)
Comment (pieter-edelman-nictiz) [reply resources/zib/zib-CareTeam.xml:130]: We don't need to, but we can populate it when appropriate. I think this is a useful addition.

File: resources/zib/zib-CareTeam.xml (hunk 4 above)
Comment (pieter-edelman-nictiz) [reply resources/zib/zib-CareTeam.xml]: Done!

File: resources/zib/zib-CareTeam.xml (hunk 2 above)
Comment (Ashwin-nictiz) [reply resources/zib/zib-CareTeam.xml:70]: Are you sure? See link: https://zibs.nl/wiki/ContactPerson-v3.4(2020EN)

Comment (pieter-edelman-nictiz) [reply resources/zib/zib-CareTeam.xml:70]: Yup. NL-CM:17.3.8 =
> CareTeamMember::Contactperson

File: resources/zib/zib-CareTeam.xml [hunk 5]
```diff
@@ -0,0 +1,172 @@
...
+      </mapping>
+    </element>
+    <element id="CareTeam.participant:patient">
+      <path value="CareTeam.participant" />
+      <sliceName value="patient" />
+    </element>
+    <element id="CareTeam.participant:patient.member">
+      <path value="CareTeam.participant.member" />
+      <short value="CareTeamMember" />
+      <definition value="The patient as member of the care_team." />
```
Comment (LuudSlagter) [inline resources/zib/zib-CareTeam.xml:94]: 'care_team' should be replaced by 'care team' (error in the zib itself).

Comment (ArdonToonstra) [reply resources/zib/zib-CareTeam.xml]: Fixed!

File: known-issues.yml [hunk 6]
```diff
@@ -22,6 +22,22 @@ zib-AddressInformation:
       - datatype: string
         reason: The mapping of zib AddressInformation on the FHIR Address datatype is the result of compatibility with HL7v3, which is the format that a lot of healthcare data in the Netherlands is stored in.  As a result of this, the zib HouseNumberIndication concept with CD datatype is mapped to a FHIR string datatype with a constraint added (the value can only be 'by' or 'to').
 
+zib-CareTeam:
+  ignored issues:
+    CareTeam.participant:contactperson.member:
+      - message: "sd-zpg-03: 'References to other resource profiles should be added next to the HL7  base references.'"
+        reason: The restiction only applies to, and defines, this slice. The slicing is open so the base references can still be used.
```
[the remark this answers is not in the document]
Comment (ArdonToonstra) [reply known-issues.yml]: Fixed!

File: resources/nl-core/nl-core-CareTeam.xml [hunk 7]
```diff
@@ -0,0 +1,82 @@
...
+        <code value="Reference" />
+        <targetProfile value="http://nictiz.nl/fhir/StructureDefinition/nl-core-Patient" />
+      </type>
+    </element>
+    <element id="CareTeam.participant:patient.onBehalfOf">
+      <path value="CareTeam.participant.onBehalfOf" />
+      <type>
+        <code value="Reference" />
+        <targetProfile value="http://hl7.org/fhir/StructureDefinition/Organization" />
+        <targetProfile value="http://nictiz.nl/fhir/StructureDefinition/nl-core-HealthcareProvider-Organization" />
```
Comment (ArdonToonstra) [inline resources/nl-core/nl-core-CareTeam.xml:58]: Klopt deze `.onBehalfOf` wel?

Comment (pieter-edelman-nictiz) [reply resources/nl-core/nl-core-CareTeam.xml:58]: Nee, die hoort bij zorgverlener te zitten.

**Comment by pieter-edelman-nictiz** [comment]
As far as I can see, all terminology errors are false negatives. The Validator is unable to resolve references on disk, there's currently no way around this. All in all this could be merged I guess.

**Comment by ArdonToonstra** [comment]
> As far as I can see, all terminology errors are false negatives. The Validator is unable to resolve references on disk, there's currently no way around this. All in all this could be merged I guess.

I thought so too. Let's merge it :)

**Review by ArdonToonstra — APPROVED** [review]

**Comment by pieter-edelman-nictiz** [comment]
Manually rebased and merged

**Pull request closed without merging**